dags/pipeline_small.py

```diff
@@ -99,6 +100,21 @@ def remove_unblurred_images(**context):
         template_searchpath=["/"],
         catchup=False,
 ) as dag:
+    @task
+    def get_config(**context):
+        workers = context['dag_run'].conf.get('workers')
```

Comment:
Wat als "workers" niet wordt meegegeven? Crasht the DAG dan?

Reply:
In dit geval is de waarde van de `workers` variabele `None` als er geen `"workers"` key bestaat in de conf dict.

`dict.get(key, default=None)` heeft een optioneel tweede argument om een standaardwaarde mee te geven wanneer de key niet bestaat. Die zou je hier ook kunnen gebuiken.